tests/test_pois.py

```diff
@@ -0,0 +1,274 @@
+import os
+import unittest
+from flask import jsonify
+from api import app, db
+from flask import Flask, request
+from api.models import POI, Media, Link, Story, StoryPOI
+import requests
+from flask import jsonify
+import json
+from datetime import date
+
+#Complete params
```

Comment:
can you elaborate more on what "complete" means here - perhaps say that "complete" pois have all required and optional attributes.

also, this is from a js styleguide, but can be applied here - comments are more readable if you have a [space before them](https://github.com/airbnb/javascript#comments--spaces)

and following the [style convention above it](https://github.com/airbnb/javascript#comments--singleline), you should have the comment right above the line of code. so no need for the newline  in between the comment and the code. this could be different for block comments, where it might make sense to have whitespace around them, but this is just a simple one line comment